fixtures/dom/src/components/Header.js

```diff
@@ -9,13 +10,12 @@ const Header = React.createClass({
     return { version, versions };
   },
   componentWillMount() {
-    fetch('https://api.github.com/repos/facebook/react/tags', { mode: 'cors' })
-      .then(res => res.json())
+    getVersionTags()
       .then(tags => {
         let versions = tags.map(tag => tag.name.slice(1));
-        versions = ['local', ...versions];
+        versions = [`local (${React.version})`, ...versions];
```

Comment:
Is this always going to be the current version you have loaded? So if you load 0.13.0, will it say `local 0.13.0`?